tsl/test/CMakeLists.txt

```diff
@@ -67,7 +67,10 @@ if (TAP_CHECKS)
     PATH="${PG_BINDIR}:$ENV{PATH}"
     PGPORT=6${TEST_PGPORT_LOCAL}
     PG_REGRESS=${PG_REGRESS}
-    ${PROVE} -I "${PG_SOURCE_DIR}/src/test/perl"
+    ${PROVE}
+    -I "${PG_SOURCE_DIR}/src/test/perl"
+    -I "${PG_LIBDIR}/pgxs/src/test/perl"
+    -I "${PG_LIBDIR}/postgresql/pgxs/src/test/perl"
```

Comment:
It does not hurt to have this in the path, but I do not understand why it is needed. Neither 13.2 nor 14beta1 place the perl modules here:
```
mats@mats-XPS-13-9380:~$ find `/usr/local/pgsql/13.2/bin/pg_config --libdir` -name '*.pm'
/usr/local/pgsql/13.2/lib/pgxs/src/test/perl/RecursiveCopy.pm
/usr/local/pgsql/13.2/lib/pgxs/src/test/perl/TestLib.pm
/usr/local/pgsql/13.2/lib/pgxs/src/test/perl/PostgresNode.pm
/usr/local/pgsql/13.2/lib/pgxs/src/test/perl/SimpleTee.pm
mats@mats-XPS-13-9380:~$ find `/usr/local/pgsql/14beta1/bin/pg_config --libdir` -name '*.pm'
/usr/local/pgsql/14beta1/lib/pgxs/src/test/perl/RecursiveCopy.pm
/usr/local/pgsql/14beta1/lib/pgxs/src/test/perl/TestLib.pm
/usr/local/pgsql/14beta1/lib/pgxs/src/test/perl/PostgresNode.pm
/usr/local/pgsql/14beta1/lib/pgxs/src/test/perl/SimpleTee.pm
```

Do you have any idea why the `postgresql/` is needed here?

Reply:
I just have it on MacOS 🤷‍♂️

Reply:
I actually don't have perl modules install on this path:

```
~ ls /home/user/dev/build/postgresql-13.2/lib/pgxs/src/test/
regress
```

Do they have to be installed manually from PosgreSQL source directory somewhere?

Reply:
You need to use `--enable-tap-tests` when running `configure` to get them installed.

So, if you have an installation that do not have the Perl modules provided, you can use the source directory to get the modules.

Reply:
I see, that's a bummer, since I have several PostgreSQL versions installed already :)

Reply:
So do I. I just rebuild them and re-install them.

```
mats@mats-XPS-13-9380:~$ ls /usr/local/pgsql/
10.2  11.6  11.8  11.9  12  12.1  12.1-nossl  12.2  12.3  12.6  13.2  14beta1  9.6.14
```